src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -48,13 +59,13 @@
     border-radius: 50%;
     z-index: z(user-avatar-status-indicator);
     display: block;
+
     @include square-size($default-indicator-size);
 
-    @each $modifier, $size in $indicator-sizes-map {
-      &--#{$modifier} {
-        @include square-size($size);
-      }
-    }
+    // NOTE: for the illusion of transparency, the border's style and color should be set by @include background-color() mixin in the appropriate ancestor
+    border-style: var(--UserAvatar__status-indicator--border-style);
+    border-color: var(--UserAvatar__status-indicator--border-color);
+    border-width: $default-indicator-border-width;
```

Comment:
There is a limitation of the ability to have transparent border that will also preclude the painting of its parent, so the border is set to a color that is set on the ancestor. Example:

Parent`div` contains component `UserAvatar` `div` which has status indicator `div`.
In order for the indicator to have the same color border as the parent, it should be passed down from the parent as a CSS --variable.

The use of the BEM style in the  variable names is deliberate in order to exclude clashes with possible other variables as well as to directly indicate by which component (or element in it) should be used.

Reply:
1) Why do you have to inherit border-style if it should always be solid
2) Why don't we just use the padding if the border is same color as the background

Reply:
> if it should always be solid

If the border is set as solid from the start, but no ancestor provides appropriate color, the indicator border will be out of sync.

The setting of the `border-style` property serves as a kind of show/hide switch but I also wanted to leave the possibility of the `border-color` mixin to use different value (or even property) as the switch, provided there's a need of variety.

>  Why don't we just use the padding if the border is same color as the background

A > B > C > D > E
E has to have the border color of A's background. Padding will expose whatever D has painted, not even necessarily D's background.

Reply:
@mike-lvov Is there going to be a "theme" background variable for each possible place `<UserAvatar>` can be used?
@sabrinaorban Is it OK to not show indicator border for the time being?

Reply:
@goran-peoski-work There probably will be not more than 2-3 themes. Light/dark theme implies that it should be sufficient to have either dark or light circle border(in any place of usage). So, let's remove that `background mixin` and merge this PR

Reply:
>  Light/dark theme implies that it should be sufficient to have either dark or light circle border(in any place of usage).

I don't see where the transparent indicator border implies usage of themes in any way. Just because there are white and black background in the example doesn't mean whichever parent the `<UserAvatar>` is put on will have a theme background, thus the question:

> Is there going to be a "theme" background variable for each possible place <UserAvatar> can be used?

@mike-lvov  How can this be ensured?

> let's remove that background mixin and merge this PR

Still waiting @sabrinaorban input to make sure it's OK to not have an indicator border.

Reply:
Yes, we can give up the avatar border!

Reply:
Removed mixin @ 8315836c8183882047216e853ad29c66567e0d2a Border will no longer be visible while the width calculation is still present for future use when appropriate color solution can be produced.